src/goto-instrument/goto_program2code.cpp

```diff
@@ -342,9 +342,11 @@ goto_programt::const_targett goto_program2codet::convert_assign_varargs(
       {this_va_list_expr});
     f.arguments().back().type().id(ID_gcc_builtin_va_list);
 
-    side_effect_expr_function_callt type_of;
-    type_of.function() =
-      symbol_exprt("__typeof__", code_typet({}, empty_typet()));
+    side_effect_expr_function_callt type_of(
```

Comment:
How do we eventually get away with this apparently-wrong type? Comment here to say where the fixup takes place?